scss/_card.scss

```diff
@@ -160,6 +160,7 @@
 // Card image
 .card-img {
   // margin: -1.325rem;
+  width: 100%;
```

Comment:
If i change your suggestion the images get distorted again (running on macbook pro in chrome)
here an example
http://codepen.io/gijsbotje/pen/OpQgXV
left is with max-width 100%, right is width 100%